### PR TITLE
Bug 1604666 - Add `payload.crashId` in telemetry.crash_v4

### PR DESCRIPTION
**Pull request by acmiyaguchi**
[bug link](https://bugzilla.mozilla.org/show_bug.cgi?id=1604666)

Checklist for reviewer:

- [x] Commits should reference a bug or github issue, if relevant
- [x] Scan the PR and verify that no changes (particularly to `.circleci/config.yml`) will cause environment variables (particularly credentials) to be exposed in test logs
- [ ] Trigger the `integration` CI test by pushing this revision [as discussed in the README](https://github.com/mozilla-services/mozilla-pipeline-schemas#packaging-and-integration-tests-optional)

For glean changes:
- [ ] Update `include/glean/CHANGELOG.md`

### CODE REVIEW

File: templates/telemetry/crash/crash.4.schema.json [hunk 1]
```diff
@@ -15,6 +15,9 @@
           "type": "string",
           "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}$"
         },
+        "crashId": {
+          "type": "string"
```
Comment (jklukas) [inline templates/telemetry/crash/crash.4.schema.json]: Is crashId expected to be a UUID? Should we enforce that with a pattern?

Comment (jklukas) [reply templates/telemetry/crash/crash.4.schema.json]: Is crashId always present, or is it potentially null?

Comment (fbertsch) [reply templates/telemetry/crash/crash.4.schema.json]: Tests for these cases would be good.

Comment (acmiyaguchi) [reply templates/telemetry/crash/crash.4.schema.json]: It does appear to be a UUID, but I'm not sure if we would start to reject any pings if we enforced the validation. I'll add this and check the integration report. 

The documentation for thing ping marks the field as optional: https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/data/crash-ping.html

The field is already included as part of the passing document here: https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/e8226ee26926df54fb641cee02c4e91b0d346a5b/validation/telemetry/crash.4.sample.pass.json#L188

Comment (acmiyaguchi) [reply templates/telemetry/crash/crash.4.schema.json]: It looks like there's a small increase in the number of rejected pings, based on the document sample:

```diff
--- 
+++ 
@@ -168,7 +168,7 @@
         "error_rate": 0.0
     },
     "telemetry.crash.4": {
-        "error_rate": 1.9
+        "error_rate": 2.0
     },
     "telemetry.deletion.4": {
         "error_rate": 0.0
```

**Comment by acmiyaguchi** [comment]
Yes, it also seems like the `crashId` is not the primary way to correlate crashes with socorro. Instead, the sha256 hash associated with the crash dump subsumes this id. This means the `crashId` field might not actually be useful for anything, so I don't think there's necessarily a case to be strict about the values in this field.

**Pull request merged**